### PR TITLE
Add Leah Garber profile pics and info

### PR DESCRIPTION
**Pull request by ldgarber**
added my profile info

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -106,7 +106,7 @@ a:visited {
 
 .student-card {
   position: relative;
-  background-image: url("../assets/img/profile-photo-placeholder.png");
+  background-image: url("../assets/img/leah-garber.png");
```
Comment (adamjonas) [inline css/styles.css]: @ldgarber This should continue to be the placeholder rather than your profile picture. Thanks!

**Comment by adamjonas** [comment]
The rest looks good. I'll merge when that line is restored.

**Comment by ldgarber** [comment]
I've replaced the error line in the css (in commit 339d642), do I need to do another pull request?

**Comment by adamjonas** [comment]
Nope that should do it. Stand by.

**Pull request merged**